src/components/Dashboard/Inventory/AdminProduct.js

```diff
@@ -72,15 +108,26 @@ export class AdminProduct extends Component {
           </button>
         </td>
         <td>
-          <button
-            id="removeButton"
-            className="admin-product__button"
-            type="button"
-            onClick={() => this.handleRemoveClick(id)}
-            disabled={remove}
-          >
-            X
-          </button>
+          {remove ? (
```

Comment:
Misleading name, as it's true meaning was explained in test description `should show undeleteButton when product is remove`
In this case maybe it should be named `removed`
so we have `product.removed = true`
`remove` is an action
`removed` is the property